content/06.prior-knowledge.md

```diff
@@ -1,27 +1,26 @@
-Knowledge graphs (KGs) which integrate related-but-different data types, provide a rich multimodal data source (e.g., Monarch Graph Database [@doi:10.1093/nar/gkw1128], hetionet [@doi:10.7554/elife.26726], PheKnowLator [@doi:10.1101/2020.04.30.071407], and the Global Network of Biomedical Relationships [@doi:10.1093/bioinformatics/bty114], Orphanet [@http://www.orpha.net]). 
+One strategy to overcome the paucity of data in rare disease is to combine a variety of data types and explore rare disease data alongside other existing knowledge.
+By using several data modalities, such as curated pathways, genetic data, or other data types, it may be possible to gain a better understanding of rare diseases.
+Knowledge graphs (KGs) which integrate related-but-different data types, provide a rich multimodal data source e.g., Monarch Graph Database [@doi:10.1093/nar/gkw1128], hetionet [@doi:10.7554/elife.26726], PheKnowLator [@doi:10.1101/2020.04.30.071407], and the Global Network of Biomedical Relationships [@doi:10.1093/bioinformatics/bty114], Orphanet [@http://www.orpha.net]. 
 These graphs connect genetic, functional, chemical, clinical, and ontological data so that relationships of data with disease phenotypes can be explored through manual review [@doi:10.1093/database/baaa015] or computational methods [@doi:10.1016/j.jbi.2021.103838, @doi:10.1142/9789811215636_0041; @doi:10.1186/s12911-019-0938-1]. (Figure[@fig:3]a)
 KGs may include links (i.e. edges) or nodes that are specific to a rare disease of interest (e.g., an FDA approved treatment would be a specific disease-compound edge in the KG) and more generalized information (e.g., gene-gene interactions noted in the literature for a different disease). (Figure [@fig:4]a)
 
-Rare disease researchers can repurpose general (i.e., not rare disease-specific) biological or chemical KGs to answer rare disease-based research questions [@doi:10.1142/9789811215636_0041] (e.g. Box 1b).
-One tactic to sift through the large amounts of data encoded in KGs is to calculate the distances between nodes of interest (e.g., diseases and drugs for Box 1b [@doi:10.1142/9789811215636_0041]); often done by determining the "embeddings" (linear representations of the position and connections of a particular point in the graph) for nodes in the KG, and calculating the similarity between these embeddings.
+Rare disease researchers can repurpose general biological or chemical KGs that are not disease-specific to answer rare disease-based research questions [@doi:10.1142/9789811215636_0041] (e.g. Box 1b).
+One tactic to sift through the large amounts of data encoded in KGs is to calculate the distances between nodes of interest (e.g., diseases and drugs for Box 1b [@doi:10.1142/9789811215636_0041]); often done by determining the "embeddings" - linear representations of the position and connections of a particular point in the graph for nodes in the KG, and calculating the similarity between these embeddings.
```

Comment:
Justin: How are these linear representations?

Reply:
Looks like i introduced the word 'linear' two years ago. I am not sure why. Best guess is I meant to convey something along the lines of a "vector representation" or a "numeric representation." 

However, I think that just removing the word "linear" is best, to avoid having to add too much of an explanation about the concept of embeddings (out of scope of this review). 

```suggestion
One tactic to sift through the large amounts of data encoded in KGs is to calculate the distances between nodes of interest (e.g., diseases and drugs for Box 1b [@doi:10.1142/9789811215636_0041]); often done by determining the "embeddings" - representations of the position and connections of a particular point in the graph for nodes in the KG, and calculating the similarity between these embeddings.
```

Reply:
I added a simple edit : https://github.com/jaybee84/ml-in-rd/pull/254/files

Reply:
LGTM. Thanks!